governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -144,7 +147,7 @@ const General = ({ proposerServerUrl }: { proposerServerUrl: string }) => {
       setExistingSymbols(new Set(Object.keys(symbolToData)))
       setData(sortDataMemo(symbolToData))
     }
-  }, [rawConfig, dataIsLoading, sortDataMemo])
+  }, [rawConfig, dataIsLoading, sortDataMemo, cluster])
```

Comment:
missing dependency `cluster` since it is being used in the `useEffect` hook

Reply:
mmmm I'm not sure about this